test/unit/play-display-icon-test.js

```diff
@@ -27,88 +27,89 @@ describe('PlayDisplayIcon', function() {
         it('should not add class if config is not set', function() {
             displayIcon = new PlayDisplayIcon(model, {}, element);
 
-            expect(icon.className).to.not.include('jw-ab-idle-');
-        });
-
-        it('should not add class if config is default', function() {
-            model.set('idleButton', 'default');
-
-            displayIcon = new PlayDisplayIcon(model, {}, element);
-
-            expect(icon.className).to.not.include('jw-ab-idle-');
+            expect(displayIcon.icon.className).to.not.include('jw-ab-idle-label');
         });
 
         it('should not add class if config is invalid', function() {
-            model.set('idleButton', 'invalid');
+            model.set('idleButtonText', 'invalid text');
 
             displayIcon = new PlayDisplayIcon(model, {}, element);
 
-            expect(icon.className).to.not.include('jw-ab-idle-');
+            expect(displayIcon.icon.className).to.not.include('jw-ab-idle-label');
```

Comment:
Since you have a test [below](https://github.com/jwplayer/jwplayer/pull/2942/files?utf8=%E2%9C%93&diff=unified#diff-4447a731931f7cb8b51990ed14e927dfR41) that checks that both the class and the txt are added, how about another assertion that confirms the txt is not added?